build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
Isn't this implicit in cargo?

Reply:
By default the build script reruns if any file in the crate changes - this turns that behavior off.